source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -358,6 +358,10 @@ def wrapper(*args, **kwargs):
                     args['allowed_id[]'] = list(allowed_ids)
                     args['_type'] = 'journeys'
                     args['rel'] = 'same_journey_schedules'
+                    # Shared section journeys shouldn't be removed in 'same_journey_schedules'
+                    if '_no_shared_section' in args:
+                        del args['_no_shared_section']
```

Comment:
I think it would fit more into `is_journey_schedule` parameter, it's designed to handle and deactivate adverse parameters : https://github.com/CanalTP/navitia/blob/dev/source/jormungandr/jormungandr/interfaces/v1/Journeys.py#L559